Formula/cppcheck.rb

```diff
@@ -36,6 +36,10 @@ def install
     # CFGDIR is relative to the prefix for install, don't add #{prefix}.
     system "make", "DESTDIR=#{prefix}", "BIN=#{bin}", "CFGDIR=/cfg", "install"
 
+    # Move the python addons to the cppcheck pkgshare folder
+    (pkgshare/"addons").install Dir.glob bin/"*.py"
+    rm Dir.glob bin/"*.py"
```

Comment:
If you've moved them all in the above step there shouldn't be anything left to `rm` right?